lms/djangoapps/django_comment_client/management/commands/rename_user.py

```diff
@@ -0,0 +1,34 @@
+from django.core.management.base import BaseCommand, CommandError
+from django.contrib.auth.models import User
+import pymongo
+from pymongo import MongoClient
+from pymongo.errors import PyMongoError
+
+class Command(BaseCommand):
+    args = '<old_username> <new_username>'
+    help = 'Modify the username of an existing user'
+
+    def handle(self, *args, **options):
+        if len(args) != 2:
+            raise CommandError('Usage is rename_user {0}'.format(self.args))
+
+        try:
+            user = User.objects.get(username=args[0])
+        except User.DoesNotExist:
+            raise LookupError("User %s does not exist. Available users: %s" % (args[0], User.objects.all()))
```

Comment:
It's a nice thought to provide hints, but I don't think we should be listing >1M db table entries.  So it may be better just to take this out